script.js

```diff
@@ -1,54 +1,75 @@
-const addBtn = document.getElementById('add');
 const booksList = JSON.parse(localStorage.getItem('books'));
+const addBtn = document.getElementById('add');
+
+class Display {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.id = Date.now();
+  }
+
+  static addBook() {
+    const Title = document.getElementById('title').value;
+    const Author = document.getElementById('author').value;
+    const list = document.getElementById('book-list');
+    const addDiv = document.createElement('div');
+    addDiv.classList.add('book');
+    const newBook = new Display(Title, Author);
+    booksList.push(newBook);
```

Comment:
* Kindly fix this error I get when I try to add a book. HINT 💭 you are trying to push an object into a variable (bookList) kindly make bookList an empty array so that you could push your newly created book details into it and also perform other array functions on them 👍 
![image](https://user-images.githubusercontent.com/52497246/191506501-8f628f6b-fd9e-4db0-8aa3-7bdab4535f98.png)